packages/app-store/_utils/useAddAppMutation.ts

```diff
@@ -57,6 +57,7 @@ function useAddAppMutation(_type: App["type"] | null, allOptions?: UseAddAppMuta
             { variant: variables && variables.variant, slug: variables && variables.slug },
             location.search
           ),
+      fromApp: true,
```

Comment:
Adding this, not sure how atoms or the api will handle OAuth integrations but if there's an error from installing an integration from the web app we should redirect back.